tests/Integration/Behaviour/Features/Scenario/Product/add_product.feature

```diff
@@ -0,0 +1,17 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags add
+@reset-database-before-feature
+Feature: Add basic product from Back Office (BO)
+  As a BO user
+  I need to be able to add new product with basic information from the BO
+
+  @add
+  Scenario: I add a product with basic information
+    When I add product "product1" with following basic information in default language:
+      | name | bottle of beer |
+      | type | standard       |
+    Then product "product1" should have following values:
+      | active           | false          |
+      | condition        | new            |
+    And product "product1" type should be standard
```

Comment:
Maybe these two checks could be done along with the previous one, with everything in a table, and only one function needed to check all product data

Reply:
kinda, but it at least type depends from couple product properties its not just "product->type". So  it would still contain a dedicated private method with some statement to identify if its tis field. e.g.
```
if ($field === 'type') {
    $this->assertTypeField();
}
```
I t seemed overhead to try merge it in this case.
So i thought we can make one merged method for  simple properties where we can check simply "if (field === $product->field)", and custom ones for more complicated props.